src/article/shared/FootnoteComponent.js

```diff
@@ -37,6 +39,21 @@ export default class FootnoteComponent extends Component {
         )
       )
     }
+
+    if (mode === METADATA_MODE) {
+      const footer = $$('div').addClass('se-footer').append(
+        $$('button').addClass('se-remove-item').append(
+          $$(FontAwesomeIcon, { icon: 'fa-trash' }).addClass('se-icon'),
+          'Remove'
```

Comment:
Please start using `this.getLabel()`.
Brother of `CopyNPaste` is `StringLiterals`.

Reply:
👍